setup.py

```diff
@@ -34,64 +33,32 @@ def run(self):
         # Despite what the setuptools docs say, the omission of this
         # in setuptools is a bug, not a feature.
         print("== Installing Nautilus Python extension")
-        src_file = "nautilus_open_any_terminal/open_any_terminal_extension.py"
+        src_file = "src/nautilus_open_any_terminal.py"
         dst_dir = os.path.join(self.install_data, "share/nautilus-python/extensions")
         self.mkpath(dst_dir)
         dst_file = os.path.join(dst_dir, os.path.basename(src_file))
         self.copy_file(src_file, dst_file)
         print("== Done!")
 
         print("== Installing language files")
-        for po_path in glob.glob(
-            str(pathlib.Path("nautilus_open_any_terminal") / PO_FILES)
-        ):
+        for po_path in glob.glob(str(pathlib.Path("nautilus_open_any_terminal") / PO_FILES)):
             src_file = pathlib.Path(po_path.replace(".po", ".mo"))
-            original_folder = os.path.dirname(src_file).replace(
-                "nautilus_open_any_terminal/", ""
-            )
+            original_folder = os.path.dirname(src_file).replace("nautilus_open_any_terminal/", "")
             dst_dir = os.path.join(self.install_data, "share", original_folder)
             self.mkpath(dst_dir)
             dst_file = os.path.join(dst_dir, os.path.basename(src_file))
             self.copy_file(src_file, dst_file)
         print("== Done!")
 
         print("== Installing GSettings Schema")
-        src_file = "./nautilus_open_any_terminal/schemas/com.github.stunkymonkey.nautilus-open-any-terminal.gschema.xml"
+        src_file = "./src/schemas/com.github.stunkymonkey.nautilus-open-any-terminal.gschema.xml"
         dst_dir = os.path.join(self.install_data, "share/glib-2.0/schemas")
         self.mkpath(dst_dir)
         dst_file = os.path.join(dst_dir, os.path.basename(src_file))
         self.copy_file(src_file, dst_file)
```

Comment:
you could use pathlib methods here and elsewhere in setup.py like [`glob`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob), [`with_suffix`](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.with_suffix), [`relative_to`](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.relative_to)

Reply:
please recheck. Did more rewriting then expected, but I like your suggestion.